101-azure-api-for-fhir/azuredeploy.parameters.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+     "serviceName":{
+        "value": "GEN-UNIQUE"
+     },
+     "location":{
```

Comment:
If this is globally available remove this - if it's not add allowedValues to the parameter in the template.